runtime/api/Robot.py

```diff
@@ -12,49 +12,537 @@
 >>> from api import Robot
 '''
 import memcache
-
 memcache_port = 12357
 mc = memcache.Client(['127.0.0.1:%d' % memcache_port]) # connect to memcache
 
+from uid_did_conversions import *
+
 motor = {}
 
+def _lookup(name): #Returns actual device ID given name
+    # TODO: HARDCODED VALUE FOR TESTING ONLY
```

Comment:
MAJOR HACK here!!!

This should be replaced with the identity function `def _lookup(name): return name`